crates/ratchet-web/src/model.rs

```diff
@@ -344,4 +381,43 @@ mod tests {
         log::warn!("Result: {:?}", result);
         Ok(())
     }
+
+    #[wasm_bindgen_test]
+    async fn whisper_browser_custom() -> Result<(), JsValue> {
+        log_init();
+        let download_cb: Closure<dyn Fn(f64)> = Closure::new(|p| {
+            log::info!("Provided closure got progress: {}", p);
+        });
+        let js_cb: &js_sys::Function = download_cb.as_ref().unchecked_ref();
+
+        let mut model = Model::load_custom(
+            "https://huggingface.co/FL33TW00D-HF/whisper-base2/resolve/main".to_string(),
```

Comment:
Is this going to work?

Reply:
Yep. That dose change base URL + path. So if you upload sane files to different location everything is going to work 

Checked that with wasm_bindgen_test

Reply:
No no all good! Sorry couldn't test as there is an issue with upstream, working on it now.

Reply:
Is there anything I can do to help?

Reply:
No it's upstream, don't worry about it i will disable CI for now.